lib/components/form/mode-buttons.tsx

```diff
@@ -6,10 +6,10 @@ import { buttonCss } from './batch-styled'
 import { ComponentContext } from '../../util/contexts'
 import { getFormattedMode } from '../../util/i18n'
 import { injectIntl, IntlProvider, IntlShape } from 'react-intl'
-import FormattedMode from '../util/formatted-mode'
 import Icon from '../util/icon'
 
 export type Mode = {
```

Comment:
I think you need to do address what the auto formatter is complaining about in this file